testing/kuttl/e2e/exporter-standby/README.md

```diff
@@ -0,0 +1,9 @@
+# Exporter connection on standby cluster
+
+The exporter standby test will deploy two clusters, one primary and one standby.
+Both clusters have monitoring enabled and are created in the same namespace to
+allow for easy connections over the network.
+
+The `ccp_monitoring` password for both clusters are updated to match allowing
+the exporter on the standby cluster to query postgres using the proper `ccp_monitoring`
+password.
```

Comment:
🤔 Would it be easier or harder to pull the password from the primary and set that for the standby instead of resetting the password for both? I'm just thinking it might save a step. But this actually tests what we want it to, so this isn't a thing to change at this point.

Reply:
It wouldn't be that difficult to grab the contents of the primary secret. But yeah, this is the actual workflow I would expect someone to use, and I don't think it adds too much complexity to the test